12LessonTwelve/in-class-project/index.html

```diff
@@ -1,10 +1,55 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Delivery Service</title>
+    <!--Google fonts-->
+    <!--Bootstrap-->
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+    <header class="herospace">
+            <h1> Order Takeaway or Delivery Food </h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+    </header>
+
+    <main class="row">
+        <section>
+            <div class="location col-sm-6 col-lg-3">
+                <img src="img/icon_home_1.svg">
+                <h2> Search by address </h2>
+                <p> Find all restaurants available in your zone.</p>
+            </div>
+
+            <div class="shop col-sm-6 col-lg-3">
+                <img src="img/icon_home_2.svg">
+                <h2> Search by address </h2>
+                <p> Find all restaurants available in your zone.</p>
+            </div>
 
+            <div class="lock col-sm-6 col-lg-3">
+                <img src="img/icon_home_3.svg">
+                <h2> Search by address </h2>
+                <p> Find all restaurants available in your zone.</p>
+            </div>
+
+            <div class="car col-sm-6 col-lg-3">

```

Comment:
Best practice in bootstrap is to start sizing your `div`s with `col-xs-` - remember that `col-xs-` width will apply to everything else - that is until you specify widths for `sm`, `md` and`lg`.